### PR TITLE
fix: DEV-3941: Enable adjusting player head position using TimeBox

### PR DESCRIPTION
**Pull request by juliosgarbi**

### CODE REVIEW

File: src/components/SidePanels/DetailsPanel/RegionEditor.tsx [hunk 1]
```diff
@@ -73,8 +73,7 @@ const RegionEditorComponent: FC<RegionEditorProps> = ({
           endTime={region.end}
           minTime={0}
           maxTime={region?._ws_region?.duration}
-          startTimeReadonly={true}
-          endTimeReadonly={true}
+          endTimeReadonly={false}
```
Comment (bmartel) [inline src/components/SidePanels/DetailsPanel/RegionEditor.tsx:76]: is there any reason this is set to false? I see the others just removed these properties.

File: src/components/TimeDurationControl/TimeBox.tsx [hunk 2]
```diff
@@ -87,23 +87,29 @@ export const TimeBox: FC<TimerProps> = ({
   const handleChangeInput = (e: React.FormEvent<HTMLInputElement>) => {
     let input = e.currentTarget.value;
 
-    input = input.replace(/[^0-9]+/g, '');
-
-    input = `${input.substr(0, 2)}:${input.substr(2, 2)}:${input.substr(4, input.length)}`;
-
-    input = input.substring(0, 9);
+    input = input.replace(/\D/g, '')
```
Comment (bmartel) [inline src/components/TimeDurationControl/TimeBox.tsx:90]: Currently this does not allow for a user to backspace a value, and type in the new value of some portion. It is really hard to accurately input the times you need to. I think we need some improvements to the user experience here.

**Review by bmartel — CHANGES_REQUESTED** [review]

File: src/components/SidePanels/DetailsPanel/RegionEditor.tsx (hunk 1 above)
Comment (juliosgarbi) [reply src/components/SidePanels/DetailsPanel/RegionEditor.tsx]: Brandon, endTimeReadonly was set to true by default, but it makes more sense for both to be false by default and we set true when we need. I already changed this!

**Comment by juliosgarbi** [comment]
wrong pr

**Pull request closed without merging**